src/routes/admin_routes.js

```diff
@@ -70,41 +68,15 @@ router.get('/publish/:id', ensureAdmin, async (req, res) => {
       `/content/${document.type}/${document.id}`,
       `${req.user.username} has approved and published your content: ${document.title}`,
     );
-  }
-
-  const smtpTransport = mailer.createTransport({
-    name: 'CubeCobra.com',
-    secure: true,
-    service: 'Gmail',
-    auth: {
-      user: process.env.EMAIL_CONFIG_USERNAME,
-      pass: process.env.EMAIL_CONFIG_PASSWORD,
-    },
-  });
 
-  const message = new Email({
-    message: {
-      from: 'Cube Cobra Team <support@cubecobra.com>',
-      to: document.owner.email,
-      subject: 'Your content has been published',
-    },
-    send: true,
-    juiceResources: {
-      webResources: {
-        relativeTo: path.join(__dirname, '..', 'public'),
-        images: true,
-      },
-    },
-    transport: smtpTransport,
-  });
+    //Normal hydration of User does not contain email, thus we must fetch it in order to notify about their application
+    const owner = await User.getByIdWithSensitiveData(document.owner.id);
```

Comment:
Hydrated user objects don't have email. In order to send the emails to the applications we must load their User with their sensitive data in order to have email